assets/vue/components/PageBlocked.vue

```diff
@@ -22,8 +12,28 @@
     <table class="table">
       <thead>
         <tr>
-          <th>Incident</th>
-          <th>Groups</th>
+          <th>
+            Incident
+            <input
+              v-model="matchText"
+              type="text"
+              class="form-control"
+              id="inlineSearch"
+              title="Partial incident# or package name are matched"
+              placeholder="Search for incident/package"
+            />
+          </th>
+          <th>
+            Groups
+            <input
+              v-model="groupNames"
+              type="text"
+              class="form-control"
+              id="inlineSearch"
```

Comment:
Unless there's some Vue magic going on that I don't see it is likely a bad idea to have multiple HTML elements with the same ID.